tests/snapshots/assets__check_file@tablex.typ-40.snap

```diff
@@ -1041,11 +1043,9 @@ snapshot_kind: text
 
 // Fetches an entire column of cells (all positions with the given x).
 #let grid-get-column(grid, x) = {
-  range(grid-count-rows(grid)).map(y => grid-at(
-    grid,
-    x,
+  range(grid-count-rows(grid)).map((
```

Comment:
We may need to better handle this case.

Reply:
And we can add an option to put the arguments on the same line, e.g. 
```typst
grid-at(
  grid, x
)
```

Reply:
I initially want to do this but I quickly figure out rustfmt never format code like this. So I decide to follow rustfmt's design